inviteme/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/gorilla/schema"
+	"net/http"
+	"net/url"
+)
+
+const (
+	inviteLink    = "https://jdsb-test-ground-slack-invite.herokuapp.com/"
+	imageLink     = "http://giphygifs.s3.amazonaws.com/media/5q7XU3MurT0pW/giphy.gif"
+	imageText     = "I came in like a wrecking ball"
+	imageAltText  = "Picture of a cat wrecking neatly stacked books"
+	linkMsg       = "Click on the link below to get an invite to *jdsb-wrecking-ball* workspace: "
+	myJdsbSlackID = "UGAA0BJ4R"
+)
+
+var (
+	decoder = schema.NewDecoder()
+)
+
+func handler(request events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+	parsedBody, err := url.ParseQuery(request.Body)
```

Comment:
You'll want to authenticate the request since we have no way of controlling function permissions via Netlify. I've provided a simple API to do that, you just need to add a line in your code like so: https://github.com/junior-dev-struggle-bus/struggle-slack-app/blob/895d67adbc49655ac422434b079a904a57cfb96c/umbrella/router.go#L192

You'll need to import the following library:
https://github.com/phoenixcoder/slack-golang-sdk/blob/master/auth/auth.go

To test in your dev environment, you'll need to add your Slack App's Signing Secret to your developer slack app: https://github.com/phoenixcoder/slack-golang-sdk/blob/de18e751112634d6dfb92c6be24536f1781427ba/auth/auth.go#L21

**Reference:**
Getting your Slack App's Signing Secret: https://api.slack.com/docs/verifying-requests-from-slack#signing_secrets_admin_page
Add environment var to your netlify: https://www.netlify.com/docs/continuous-deployment/#environment-variables